src/activities/multiplication-chart/MultiplicationChart.qml

```diff
@@ -0,0 +1,287 @@
+                           PropertyChanges { target: dotsGridCol; clickedFlagCol: false}
+                       },
+                       State {
+                           name: "active"
+                           PropertyChanges { target: dotsGridCol; color: "red"}
+                           PropertyChanges { target: dotsGridCol; clickedFlagCol: false}
+                       }
+                   ]
+
+                   GCText{ text: index}
+
+                   MouseArea {
+                       anchors.fill: parent
+                       onClicked: {
+                           if (dotsGridCol.state == "default") {
+                               dotsGridCol.state = "active"
+                               items.colSelected = index
+                           }
+                           else {
+                               dotsGridCol.state = "default"
+                               items.colSelected = 0
+                           }
+                           Activity.makeOtherRowInColWhite()
+                           Activity.changesInMainBoard()
+                       }
+                   }
+                }
+            }
+
+        }
+
+        Grid {
+            id: grid
+            anchors {
+                top: gridRow.bottom
+                topMargin: items.spacing
+                left: gridCol.right
+                leftMargin: items.spacing
+            }
+
+            spacing: items.spacing
+            columns: 10
+            rows: 10
+
+            Repeater {
+                id: repeater
+                model: 100
+                Rectangle {
+                    id: dots
+                    width: items.rectWidth
+                    height: items.rectHeight
+                    color: "green"
+
+                    GCText {
+                        text: (Math.floor(index/10) + 1) * (index%10 + 1)
+                    }
+
+                }
+            }
+        }
+
+        GCText{
+            id: instruction
+            anchors {
+                top: parent.top
+                topMargin: main.height/3 - 2*instruction.height/3
+                right: parent.right
+                rightMargin: (main.width/9)
+            }
+            fontSize: regularSize
+            text: qsTr("Instruction:\n")+
+                    qsTr("   Multiplicand x Multiplier = Answer\n")+
+                    qsTr("       1) Select the Column\n")+
+                    qsTr("       2) Select the Row\n")+
+                    qsTr("       3) State the answer\n")+
+                    qsTr("       4) Press Enter")
+        }
+
+        GCText {
+            id: question
+            anchors {
+                horizontalCenter: instruction.horizontalCenter
+                horizontalCenterOffset: -30
+                top: instruction.bottom
+                topMargin: 40
+            }
+            fontSize: hugeSize
+            text: items.rowQues + " X " + items.colQues + " = "
+        }
+
+        TextInput {
```

Comment:
We have a Numpad item in core that could be better than having a TextInput, what do you think?